tests/third_party/cupy/math_tests/test_arithmetic.py

```diff
@@ -26,7 +26,12 @@
         testing.product(
             {
                 "nargs": [1],
-                "name": ["reciprocal", "angle"],
+                "name": [
```

Comment:
below tests are started failing now, could you please check or probably mute them:
```
tests/third_party/cupy/math_tests/test_arithmetic.py::TestArithmeticRaisesWithNumpyInput_param_3_{name='angle', nargs=1}::test_raises_with_numpy_input
tests/third_party/cupy/math_tests/test_arithmetic.py::TestArithmeticUnary_param_29_{arg1=2.0, name='reciprocal'}::test_unary
```

Reply:
Since Iadded new parameters to the test, I had to update the name of tests (`param_*` part) in the skipped files.